pagopa/io-dev-api-server
# chore: [IOBP-691] Add pre-selected psp list filter for the calculate fees request

## Short description
This PR adds the PSP pre-selected filter logic for the calculateFees request

## List of changes proposed in this pull request
- If the calculateFees request receives a `idPspList`, it will filter from the available PSP list the PSP with the same id provided from the list.

## How to test
- Checkout this app IO branch https://github.com/pagopa/io-app/pull/5838
- When you start a payment with the "Pagamento veloce PayPal" you should be able to navigate directly to the payment confirmation screen
